content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -15,36 +15,36 @@ test: yes
 ---
 
 [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
-offer three different options:
+describes several ways to configure liveness and readiness probes including:
 
 1. Command
-1. TCP request
 1. HTTP request
 
-This guide shows how to use these approaches in Istio with mutual TLS enabled.
+The command approach works with Istio regardless of whether or not mutual TLS is enabled.
 
-Command and TCP type probes work with Istio regardless of whether or not mutual TLS is enabled. The HTTP request approach requires different Istio configuration with
-mutual TLS enabled.
+The HTTP request approach, on the other hand, requires special Istio configuration when mutual TLS is enabled.
+This is because the health check requests to the `liveness-http` service are sent by Kubelet,
+which does not have an Istio issued certificate. Therefore when mutual TLS is enabled,
+the health check requests will fail.
 
-## Before you begin
+Istio solves this problem by rewriting the application `PodSpec` readiness/liveness probe,
+so that the probe request is sent to [Pilot agent](/docs/reference/commands/pilot-agent/).
+Pilot agent then redirects the request to the application, strips the response body, only returning the response code.
 
-* Understand [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), Istio
-[authentication policy](/docs/concepts/security/#authentication-policies) and [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
+This feature is enabled by default in all built-in Istio [configuration profiles](/docs/setup/additional-setup/config-profiles/)
+but can be disabled as described below.
 
-* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled.
+## Liveness and readiness probes using the command approach
 
-## Liveness and readiness probes with command option
+Istio provides a [liveness sample]({{< github_file >}}/samples/health-check/liveness-command.yaml) that
+implements this approach. To demonstrate it working with mutual TLS enabled, first configure an authentication policy
+and a destination rule:
```

Comment:
Think DR is not needed any more with auto mTLS.

Reply:
Good point. What is this DR doing, esp. given that it has `host: "*.default.svc.cluster.local"`, so it has nothing to do with the liveness sample, running in the istio-io-health ns. I think I'll just remove it.